pkg/x25519_chacha20poly1305.js

```diff
@@ -361,6 +361,11 @@ module.exports.__wbindgen_is_string = function(arg0) {
     return ret;
 };
 
+module.exports.__wbg_msCrypto_6e7d3e1f92610cbb = function(arg0) {
```

Comment:
Assuming this was just wasm-pack moving something around? I don't think it's my listing / formatting. Just want to confirm with one of y'all. I'm still a bit new to rust/wasm tooling

Reply:
yeah this is the build which again is unusual to have in the repo. I wouldn't worry much about the build files in `/pkg`